README.md

```diff
@@ -48,17 +48,19 @@ Make sure you have the xcode command line tools installed. This can be done by r
 
 #### Ubuntu 18.04, Debian 9, Zorin 15
 
-`sudo apt install pkg-config build-essential libgtk-3-dev libwebkit2gtk-4.0-dev`
+`sudo apt install libgtk-3-dev libwebkit2gtk-4.0-dev`
```

Comment:
pkg-config and build-essential are needed for gcc - at least in Ubuntu 16.04. Has that changed?

Reply:
nope, it is 100% needed. I removed it for consistency. For the rest of the distros we only suggest webkit libs.
minor thing. but can be potentially confusing. I have no strong opinion this was just a suggestion. proceed however you feel.